modules/microshift-olm-deploy-ops-spec-ns.adoc

```diff
@@ -0,0 +1,251 @@
+//Module included in the following assemblies:
+//
+//* microshift_running_apps/microshift-operators-olm.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-OLM-deploy-Operators-specific-namespace_{context}"]
+= Adding OLM-based Operators to a networked cluster in a specific namespace
+
+Use this procedure if you want to specify a namespace for an Operator, for example, `olm-microshift`. In this example, the catalog is scoped and available in the global `openshift-marketplace` namespace. The Operator uses content from the global namespace, but runs only in the `olm-microshift` namespace. For {microshift-short} clusters that have network connectivity, Operator Lifecycle Manager (OLM) can access sources hosted on remote registries.
+
+[IMPORTANT]
+====
+All of the Operators installed in a specific namespace must have the same watch scope. In this case, the watch scope is *OwnNamespace*.
+====
+
+.Prerequisites
+* The {oc-first} is installed.
+* Operator Lifecycle Manager (OLM) is installed.
+* You have created a custom catalog that is running in the global namespace.
+
+.Procedure
+
+. Confirm that OLM is running by using the following command:
++
+[source,terminal]
+----
+$ oc -n openshift-operator-lifecycle-manager get pod -l app=olm-operator
+----
++
+.Example output
+[source,terminal]
+----
+NAME                           READY   STATUS    RESTARTS   AGE
+olm-operator-85b5c6786-n6kbc   1/1     Running   0          16m
+----
+
+. Confirm that the OLM catalog Operator is running by using the following command:
++
+[source,terminal]
+----
+$ oc -n openshift-operator-lifecycle-manager get pod -l app=catalog-operator
+----
++
+.Example output
+[source,terminal]
+----
+NAME                                READY   STATUS    RESTARTS   AGE
+catalog-operator-5fc7f857b6-tj8cf   1/1     Running   0          16m
+----
+
+. Create a namespace by using the following example YAML:
++
+.Example namespace YAML
+[source,YAML]
+----
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: olm-microshift
+----
++
+. Apply the namespace configuration using the following command:
++
+----
+$ oc apply -f _<ns.yaml>_ <1>
+----
+<1> Replace _<ns.yaml>_ with the name of your namespace configuration file. In this example, `olm-microshift` is used.
++
+.Example output
+[source,terminal]
+----
+namespace/olm-microshift created
+----
+
+. Create the Operator group YAML by using the following example YAML:
++
+.Example Operator group YAML
+[source,yaml]
+----
+kind: OperatorGroup
+apiVersion: operators.coreos.com/v1
+metadata:
+  name: og
+  namespace: olm-microshift
+spec: <1>
+  targetNamespaces:
+  - olm-microshift
+----
+<1> Omit the `spec.targetNamespaces` field and values for Operators using the global namespace.
```

Comment:
Reword as: "For Operators using the global namespace, omit the `spec.targetNamespaces` field and values."